std_msgs/Header header
string label
geometry_msgs/Pose pose
bool detected